syne_tune/optimizer/baselines.py

```diff
@@ -31,12 +30,26 @@
 from syne_tune.optimizer.schedulers.transfer_learning import (
     TransferLearningTaskEvaluations,
 )
+from syne_tune.optimizer.schedulers.random_seeds import RandomSeedGenerator
 from syne_tune.try_import import (
     try_import_blackbox_repository_message,
     try_import_bore_message,
+    try_import_botorch_message,
 )
 
 
+def _random_seed_from_generator(random_seed: int) -> int:
```

Comment:
To explain: Inside FIFOScheduler, random_seed is used to create a seed generator, which is used to obtain seeds for different components. The first one is the random_state of the searcher.

This works only if the searcher is created inside FIFOScheduler construction (if the `searcher` argument is a string). So, the helper here is for cases where the searcher object is created up front.

An alternative would be to use the same seed for the searcher and all the FIFOScheduler components. But that results in bad behaviour, such as the random configs being sampled to optimize the acquisition function always include all the configs which were suggested already before.